pages/search.js

```diff
@@ -179,4 +179,4 @@ function SearchPage() {
   );
 }
 
-export default withData(SearchPage);
+export default withApollo()(SearchPage);
```

Comment:
Leaving out `{ssr: true}` in `withApollo` here will cause next.js's  `useRouter` returning `{}` for `router.query` on server-side rendering & first render on client, because next.js will think the page [does not meet data fetching requirements](https://nextjs.org/docs/api-reference/next/router#router-object).

It's valid for search page because this page is not meant to be accessed directly nor by search engines.